2023/Fortune_Obadiah/README.md

```diff
@@ -12,9 +12,96 @@ I am enthusiastic and fully prepared to roll up my sleeves and immerse myself in
+- [ ] Write a short blog post which demonstrates your expertise in **technical writing**. In English, with at least one image, less than 500 words. Demonstrate your expertise _based on past experiences_. For example, any specific documentation writing you’ve done in the past, or other technical writing you’ve made. The more specific the better. The post has to be posted on a publicly-available platform (for example Dev.to, Hashnode, Medium, your own website, anywhere public), and you must also add it as a new section in your personal file in this repository (so we can provide feedback on the contents).
+- [ ] Ask [@thibaudcolas](https://github.com/thibaudcolas) via email or on Slack to give you editor access to a Wagtail website. Once you’ve received access, we ask you to create a new page demonstrating some of Wagtail’s accessibility features.
+- [ ] With your access to the demo site and by reviewing our [user guide](https://guide.wagtail.org/), identify a feature of Wagtail which isn’t documented, and report it back on the [guide project issue tracker](https://github.com/wagtail/guide/issues). Make sure to explain what the feature is, why and how you think it should be documented.
+- [ ] Draft the new content to document the feature of Wagtail which you’ve reported isn’t documented. Add it to the issue you created as a new comment, so we can provide feedback.
+- [ ] Ask [@thibaudcolas](https://github.com/thibaudcolas) via email or on Slack to give you editor access to [guide.wagtail.org](https://guide.wagtail.org/). Once you’ve received access, we ask you to make the necessary changes in the CMS to add the feature documentation.
+- [ ] TBC – Additional tasks specific to the Outreachy application process.
+
+##  Research 
+
+### Understanding CMS
+
+[What is Content Management System](https://blog.golayer.io/business/content-management-system-cms)
+
+[Knowing More about CMS](https://sitechecker.pro/cms-functions-and-definition/)
+
+### Wagtail As A Project
+
+[Wagtail Website](https://wagtail.org/about-wagtail/)
+
+[Made With Wagtail](https://madewithwagtail.org/)
+
+[Authorising Tool Accessibility Guideline](https://www.w3.org/TR/ATAG20/)
+
+[Wagtail User Guide](https://guide.wagtail.org/en-latest/)
+
+
+### Accessibility Features Documentation
+
+- [Accessibility features documentation](#accessibility-features-documentation)
+
+## First Blog Post(Research)
+[My First Journey In Open Source And Git: Overcoming Fear And Embracing Growth](https://medium.com/@fortuneweb3info/navigating-my-first-journey-in-open-source-and-git-overcoming-fear-and-embracing-growth-7f1167f5a6d8)
+
+## Second  Blog Post(Acessibility Features Docuentation)
+[Unlocking Digital Inclusion: The Quest for Accessibility](https://fortuneweb3info.medium.com/unlocking-digital-inclusion-the-quest-for-accessibility-a0a36e99e4b6)
```

Comment:
Your first image doesn’t have alt text, which is a big no-no for something like this. Please familiarise yourself with accessibility requirements when writing content on the web.

Aside from your point about Medium, this is all very generic information, which I don’t think does much to _Demonstrate your understanding based on past experiences_.

Reply:
Thank you for the corrections will do a retouch on it 👐🏾  @thibaudcolas

Reply:
I have added an alt text to blog 1 @thibaudcolas